osxkeychain/osxkeychain.go

```diff
@@ -140,7 +139,7 @@ type IdentityWithRefModel struct {
 // FindAndValidateIdentity ...
 //  IMPORTANT: you have to C.CFRelease the returned items (one-by-one)!!
 //             you can use the ReleaseIdentityWithRefList method to do that
-func FindAndValidateIdentity(identityLabel string, isFullLabelMatch bool) ([]IdentityWithRefModel, error) {
+func FindAndValidateIdentity(identityLabel string, isFullLabelMatch bool) (*IdentityWithRefModel, error) {
 	foundIdentityRefs, err := FindIdentity(identityLabel, isFullLabelMatch)
```

Comment:
if you don't allow partial match / glob match anymore then don't accept a `isFullLabelMatch` param and call `FindIdentity` with `isFullLabelMatch = true` directly